wallet/db.c

```diff
@@ -481,6 +481,84 @@ static struct migration dbmigrations[] = {
     {SQL("UPDATE forwarded_payments SET received_time=0 WHERE received_time IS NULL;"),
 	 NULL},
     {SQL("ALTER TABLE invoices ADD COLUMN features BLOB DEFAULT '';"), NULL},
+   /* We can now have multiple payments in progress for a single hash, so
+    * add two fields; combination of payment_hash & parallel_id is unique. */
+    {SQL("ALTER TABLE payments RENAME TO temp_payments;"), NULL},
+    {SQL("CREATE TABLE payments ("
```

Comment:
Should we maybe go the extra mile and call this `payment_parts` instead? We could then pull the common fields into a separate `payments` table. Would be a nice cleanups and fix that naming nit about pays being the combination of multiple payments.

Reply:
I'm not sure that should be done at the db level; it may be premature optimization?

Reply:
Yes, you're right, that'd be a bigger refactor that we should probably be done during a bit of downtime. It'd just be nice to have an actual `payment` concept that collects all the `attempt`s that are performed trying to complete it. It may be my OOP/ORM past that is catching up with me :wink: